tests/back/Pim/Enrichment/Specification/Component/Product/Updater/Converter/StringToStringDataConverterSpec.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Specification\Akeneo\Pim\Enrichment\Component\Product\Updater\Converter;
+
+use Akeneo\Pim\Enrichment\Component\Product\Updater\Converter\StringToStringDataConverter;
+use Akeneo\Pim\Enrichment\Component\Product\Value\ScalarValue;
+use Akeneo\Pim\Structure\Component\AttributeTypes;
+use Akeneo\Pim\Structure\Component\Model\Attribute;
+use Akeneo\Pim\Structure\Component\Model\AttributeInterface;
+use PhpSpec\ObjectBehavior;
+
+class StringToStringDataConverterSpec extends ObjectBehavior
+{
+    function it_is_a_simple_data_converter()
```

Comment:
```suggestion
    function it_is_initializable()
```

Miss `it_is_a_value_data_converter` function